apps/WyndDEX/app.json

```
@@ -0,0 +1,11 @@
+{
+  "appCategory": "DeFi",
+  "appName": "WyndDEX",
```

I guess "Wynd DAO" would be the correct app name, no?

```suggestion
  "appName": "Wynd DAO",
```

---

I think the community behind the project is WyndDAO, and their app is called WyndDEX. 
source: https://medium.com/wynddao/wynd-dex-a-week-one-analysis-9da3a7d0d56f

---

IMO, It depends on what the team wants. they've been rebranded as Wynd DAO since March so I guess that they want to show their app as Wynd DAO.

---

hmm haven't talked to the team, just wanted to submit the app on keplr dashboard haha. guess whichever is fine with me